src/main/java/frc/robot/commands/AutoAlignAmp.java

```diff
@@ -0,0 +1,90 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.AutoRotateUtil;
+import frc.robot.Constants;
+import frc.robot.subsystems.Limelight;
+import frc.robot.subsystems.Swerve;
+
+public class AutoAlignAmp extends Command{
+
+    private PIDController yTranslationPidController;
+    private PIDController xTranslationPidController;
+    private PIDController rotationPidController;
+
+    private Swerve s_Swerve;
+    private Limelight s_Limelight;
+    private AutoRotateUtil autoUtil;
+
+    public AutoAlignAmp() {
+        this.s_Limelight = Limelight.getInstance();
+        this.s_Swerve = Swerve.getInstance();
+        this.autoUtil = new AutoRotateUtil(s_Swerve, 0);
+        addRequirements(s_Swerve, s_Limelight);
+        
+
+        // creating yTranslationPidController and setting the toleance and setpoint
+        yTranslationPidController = new PIDController(0, 0, 0);
+        yTranslationPidController.setTolerance(1);
+        yTranslationPidController.setSetpoint(0);
+        
+        // creating xTranslationPidController and setting the toleance and setpoint
+        xTranslationPidController = new PIDController(0, 0, 0);
+        xTranslationPidController.setTolerance(0);
+        xTranslationPidController.setSetpoint(0);
+        
+        // creating rotationPidController and setting the toleance and setpoint
+        rotationPidController = new PIDController(0, 0, 0);
+        rotationPidController.setTolerance(2);
+        rotationPidController.setSetpoint(0);
+
+        // puts the value of P,I and D onto the SmartDashboard
+        // Will remove later
+        SmartDashboard.putNumber("P", 0);
+        SmartDashboard.putNumber("i", 0);
+        SmartDashboard.putNumber("d", 0);
+        //THESE VALUES WILL NEED TO BE MESSED WITH, 0 FOR NOW
+    }
+
+
+    @Override
+    public void execute() {
+        // gets value of P,I and D from smartdashboard
+        // will be removed once done tweaking
+        double kP = SmartDashboard.getNumber("P", 0);
+        double kI = SmartDashboard.getNumber("i", 0);
+        double kD = SmartDashboard.getNumber("d", 0);
+        
+        // sets the PID values for the PIDControllers
+        yTranslationPidController.setPID(kP, kI, kD);
+        xTranslationPidController.setPID(kP, kI, kD);
+        //rotationPidController.setPID(rotationkP, rotationkI, rotationkD);
+
+        double xValue = s_Limelight.getX(); //gets the limelight X Coordinate
+        double areaValue = s_Limelight.getArea(); // gets the area percentage from the limelight
+        double angularValue = s_Limelight.getSkew(); 
+
+        SmartDashboard.putNumber("Xvalue", xValue);
+        SmartDashboard.putNumber("Areavalue", areaValue);
+        SmartDashboard.putNumber("AngularValue", angularValue);
```

Comment:
we aren't using angularValue anywhere and this can be removed or commented out. Also number 73 - 75 is getting skew values we aren't using/ won't be using "probably"

Reply:
did that here:
https://github.com/CSE-FortOsageCTC/5801_Inspire_2024/pull/26/commits/90ce5b23d15331d66116ab3f8639e61aa9511d54